test/e2e_tests/specs/Reply/reply.spec.ts

```diff
@@ -0,0 +1,353 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {Browser} from '@playwright/test';
+
+import {getUser, User} from 'test/e2e_tests/data/user';
+import {PageManager} from 'test/e2e_tests/pageManager';
+import {ConfirmModal} from 'test/e2e_tests/pageManager/webapp/modals/confirm.modal';
+import {test as baseTest, expect} from 'test/e2e_tests/test.fixtures';
+import {getAudioFilePath, getTextFilePath, getVideoFilePath, shareAssetHelper} from 'test/e2e_tests/utils/asset.util';
+import {getImageFilePath} from 'test/e2e_tests/utils/sendImage.util';
+import {removeCreatedUser} from 'test/e2e_tests/utils/tearDown.util';
+import {createGroup, loginUser} from 'test/e2e_tests/utils/userActions';
+
+const test = baseTest.extend<{userA: User; userB: User}>({
+  userA: async ({api}, use) => {
+    const userA = getUser();
+    await api.createPersonalUser(userA);
+    await use(userA);
+    await removeCreatedUser(api, userA);
+  },
+  userB: async ({api}, use) => {
+    const userB = getUser();
+    await api.createPersonalUser(userB);
+    await use(userB);
+    await removeCreatedUser(api, userB);
+  },
+});
+
+const createPagesForUser = async (browser: Browser, user: User, options?: {openConversationWith?: User}) => {
```

Comment:
This approach can be shared in a common refactoring. We create pages for multiple users in a lot of test cases :)